.vscode/launch.json

```diff
@@ -735,14 +733,12 @@
       "name": "Tests: mds-jurisdiction-api",
       "type": "node",
       "request": "launch",
-      "program": "${workspaceFolder}/node_modules/mocha/bin/_mocha",
-      "args": [
-        "--project",
-        "${workspaceFolder}/tsconfig.json",
-        "-r",
-        "ts-node/register",
-        "--timeout",
-        "0"
+      "runtimeArgs": [
+        "--inspect-brk",
+        "${workspaceRoot}/node_modules/jest/bin/jest",
+        "--runInBand",
+        "--coverage",
+        "false"
```

Comment:
Why no coverage here? Would it be too low to be worth turning it on / setting a value in jest config?